### PR TITLE
update CAPV's meeting time

### PR DESCRIPTION
**Pull request by yastij**
Signed-off-by: Yassine TIJANI <ytijani@vmware.com>

This PR updates CAPV's time meeting per the poll we had, see https://kubernetes.slack.com/archives/CKFGK3SSD/p1592411607313300

### CODE REVIEW

File: sig-cluster-lifecycle/README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ The following [subprojects][subproject-definition] are owned by sig-cluster-life
 - **Contact:**
   - Slack: [#cluster-api-vsphere](https://kubernetes.slack.com/messages/cluster-api-vsphere)
 - **Meetings:**
-  - Cluster API vSphere meeting: [Tuesdays at 11:00 PT (Pacific Time)](https://zoom.us/j/875399243) (biweekly starting Wednesday March 31st, 2020). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Pacific%20Time%29).
+  - Cluster API vSphere meeting: [Thursdays at 10:00 PT (Pacific Time)](https://zoom.us/j/875399243) (biweekly starting Thursday June 24th, 2020). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
```
Comment (neolit123) [inline sig-cluster-lifecycle/README.md:119]: June 25th is a Thursday

**Comment by neolit123** [comment]
SIG calendar updated.

File: sig-cluster-lifecycle/README.md (hunk 1 above)
Comment (jdumars) [reply sig-cluster-lifecycle/README.md:119]: wow, good catch! Happy to LGTM or approve once this is fixed. @yastij

Comment (yastij) [reply sig-cluster-lifecycle/README.md:119]: whoops, good catch, updated.

**Comment by yastij** [comment]
@jdumars @neolit123 - should be good to go

**Comment by nikhita** [comment]
/approve

Leaving lgtm to @neolit123

**Pull request merged**